examples/bond/scripts/run/createBuyback.js

```diff
@@ -17,7 +17,11 @@ exports.createBuyback = async function (deployer, managerAcc, n) {
 		TMPL_APP_MANAGER: managerAcc.addr,
 		TMPL_BOND: bondToken,
 	};
-	const buybackLsig = await deployer.loadLogicByFile("buyback-lsig.py", scInitParam);
+	const buybackLsig = await deployer
+		.loadLogicByFile("buyback-lsig.py", scInitParam)
+		.catch((error) => {
```

Comment:
I think this will auto throw error. we don't need catch like this I think. Please check again

Reply:
yes it will throw, I made the changes some places but didn't in some, will remove it